app/steps/pafs_core/coastal_erosion_protection_outcomes_step.rb

```diff
@@ -131,7 +131,7 @@ def setup_coastal_erosion_protection_outcomes
       #   previous years
       #   current financial year to :project_end_financial_year
       years = [-1]
-      years.concat((2015..project_end_financial_year).to_a)
+      years.concat((2022..project_end_financial_year).to_a)
```

Comment:
Assuming this will fall under the proposed change to avoid hard-coded year values in the application.